dandiapi/api/migrations/0001_initial.py

```diff
@@ -227,35 +254,86 @@ class Migration(migrations.Migration):
                         auto_now=True, verbose_name='modified'
                     ),
                 ),
-                (
-                    'blob',
-                    models.ForeignKey(
-                        on_delete=django.db.models.deletion.CASCADE,
-                        related_name='assets',
-                        to='api.AssetBlob',
-                    ),
-                ),
-                (
-                    'metadata',
-                    models.ForeignKey(
-                        on_delete=django.db.models.deletion.CASCADE,
-                        related_name='assets',
-                        to='api.AssetMetadata',
-                    ),
-                ),
-                (
-                    'version',
-                    models.ForeignKey(
-                        on_delete=django.db.models.deletion.CASCADE,
-                        related_name='assets',
-                        to='api.Version',
-                    ),
-                ),
+                ('metadata', models.JSONField(default=dict)),
+                ('name', models.CharField(max_length=300)),
             ],
-            options={
-                'ordering': ('-modified', '-created'),
-                'get_latest_by': 'modified',
-                'abstract': False,
-            },
+        ),
+        migrations.AddIndex(
+            model_name='versionmetadata',
+            index=django.contrib.postgres.indexes.HashIndex(
+                fields=['metadata'], name='api_version_metadat_1146a6_hash'
+            ),
+        ),
+        migrations.AddIndex(
+            model_name='versionmetadata',
+            index=django.contrib.postgres.indexes.HashIndex(
+                fields=['name'], name='api_version_name_ed2a83_hash'
```

Comment:
I declared this index with this code:
```
class VersionMetadata:
    class Meta:
        indexes = [
            HashIndex(fields=['metadata']),
            ...
        ]
    ...
```
Since all indexes need to be named in Django, it generates names for you. Looking at https://github.com/django/django/blob/main/django/db/models/indexes.py#L162, it appears that `ed2a83` is a hash of the table name, the columns in the table, and the type of the hash.